pkg/app/rds_postgres.go

```diff
@@ -131,6 +130,17 @@ func (pdb *RDSPostgresDB) Install(ctx context.Context, ns string) error {
 		return err
 	}
 
+	pdb.testWorkloadName = fmt.Sprintf("%s-workload", pdb.name)
+
+	testDeployment := bastionWorkload(ctx, pdb.testWorkloadName, "postgres", pdb.namespace)
+	_, err = pdb.cli.AppsV1().Deployments(pdb.namespace).Create(context.Background(), testDeployment, metav1.CreateOptions{})
+	if err != nil {
+		return errors.Wrapf(err, "Failed to create test deployment %s", pdb.testWorkloadName)
+	}
```

Comment:
We can just create a Pod instead of deploy as this is just for debug/test purpose. Deployment is fine too